storage/shard.go

```diff
@@ -446,6 +457,11 @@ func (s *dbShard) tickAndExpire(
 		s.purgeExpiredSeries(expired)
 	}
 
+	minSleep := shardMinimumTickPerSeriesSleeps * tickSleepPerSeries
```

Comment:
having a hard time grokking what "shardMinimumTickPerSeriesSleeps" is

Reply:
Also generally confused about this code, shouldn't you just be sleeping by max(shardMinimumTickPerSeriesSleepDuration, tickSleepPerSeries)? Why the multiplication

Reply:
I'm removing this concept, yeah.